### PR TITLE
Make cwd required

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
This is related to the spec change we made to make cwd required.

**Comment by crosbymichael** [comment]
@mrunalp maybe some of the tests need the template updated with a cwd to fix the test failures.

File: start.go [hunk 1]
```diff
@@ -108,6 +108,10 @@ func startContainer(context *cli.Context, spec *specs.LinuxSpec, rspec *specs.Li
 	// ensure that the container is always removed if we were the process
 	// that created it.
 	defer destroy(container)
+
+	if spec.Process.Cwd == "" {
```
Comment (crosbymichael) [inline start.go]: What do you think about moving this validation to `loadSpec()`?

Comment (mrunalp) [inline start.go]: Sure, will do.

**Comment by mrunalp** [comment]
Updated wit the tests fixed. There is some issue on jenkins, the tests aren't even run.

**Comment by hqhq** [comment]
LGTM if janky is happy.

File: spec.go [hunk 2]
```diff
@@ -290,6 +291,15 @@ var mountPropagationMapping = map[string]int{
 	"":         syscall.MS_PRIVATE | syscall.MS_REC,
 }
 
+// validateSpec validates the fields in the spec
+// TODO: Add validation for other fields where applicable
+func validateSpec(spec *specs.LinuxSpec, rspec *specs.LinuxRuntimeSpec) error {
+	if spec.Process.Cwd == "" {
```
Comment (duglin) [inline spec.go]: if https://github.com/opencontainers/specs/pull/312 is adopted before this PR is merged, can you add a check to ensure its an abs path?  `filepath.IsAbs()`

Comment (mrunalp) [inline spec.go]: Sure.

Comment (duglin) [inline spec.go]: https://github.com/opencontainers/specs/pull/312 is merged.

Comment (duglin) [inline spec.go]: oops sorry  - no its not - will erase comment

**Comment by mrunalp** [comment]
@jfrazelle ping; Something wrong with the jenkins server.

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**